app/mailers/user_mailer.rb

```diff
@@ -1,12 +1,17 @@
-require 'pry'
 class UserMailer < ActionMailer::Base
   default :from => "application_name@domain.com"
 
-  def registration_confirmation(user)
-    mail(:to => "#{user.first_name} <#{user.email}>", :subject => "Registration Confirmation for Awesome App", :body => "http://localhost:9393/#{user.confirm_token}/confirm_email/")
-  # mail(:to => "#{user.first_name} <#{user.email}>", :subject => "Registration Confirmation for Awesome App") do |format|
-  #     format.text
-  #     format.html
-  #   end
+  def registration_confirmation(request, user)
```

Comment:
Passing in the `Request` object, so we can extract host/port - you use shotgun (port 9393) and I the default WEBrick because I really don't care (port 4567). This allows either and still passed proper host/port through